jack-sys/build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    pkg_config::probe_library("jack").unwrap();
```

Comment:
It might be a good idea to not always fail hard when the pkg-config call fails, e.g.:
```suggestion
    let _ = pkg_config::probe_library("jack");
```

Reply:
Why?

Reply:
basically every scenario in which `pkg-config` or the `jack.pc` file is unavailable, and/but the jack runtime libraries are available, as `pkg-config` isn't really necessary for the library code to work, the code doesn't access need to reference any include files or such which would be found that way, it is only used to increase the accuracy of the linking (which may be incomplete, as noted above).

Reply:
additional, it is really likely to just break completely on windows, as it is even more likely that `jack.pc` files or `pkg-config` are just unavailable there. `pkg_config::probe_library` prints a warning in that case anyway, so if the final linking is failing or does something unexpected the user has enough information to fix it, but I don't think that `pkg-config` should be always required.

Reply:
Okay, I changed this and added a comment. I have no idea how Cargo locates C libraries on Windows.